docs/DafnyRef/Options.txt

```diff
@@ -192,6 +192,9 @@ Usage: dafny [ option ... ] [ filename ... ]
   
   ---- Verification options -------------------------------------------------
   
+  /allowAxioms:<value>
```

Comment:
As per offline discussion, let's just make this default to true for now, and we'll switch it to false for Dafny 4.0

Reply:
I would have been OK with introducing this default even if we weren't doing a major version bump. I think it's OK to have minor breaking changes like this be part of minor version bumps, given the current level of maturity of Dafny.

However, since the tests are setting the value anyways, there's no effort in changing the default.